docs/kratos/self-service/flows/user-logout.mdx

```diff
@@ -3,115 +3,85 @@ id: user-logout
-selfservice:
-  flows:
-    logout:
-      after:
-        default_browser_return_url: http://test.ory.sh:4000/
+```mdx-code-block
+<CodeBlock language="ts" title="index.ts">{browserExample}</CodeBlock>
 ```
 
-<CodeTabs items={browserFlow} />
-
-If an error occurs, the browser is redirected to the [Error UI](./user-facing-errors.md).
-
-## Self-service logout for client-side browser applications
+### After logout redirect
 
-Similar to server-side browser applications, Ory Identities first generates a Logout URL for a given Ory Session Cookie. However,
-you can simply call the Logout URL using an AJAX request. Ory Identities returns a `204 No Content` response on success or an
-error otherwise.
+After successful logout, the browser will be redirected either to the `return_to` query parameter from the initial request URL, or
+fall back to the default post logout return URL.
 
-import SelfServiceLogoutSPAFlow from "@site/src/theme/SelfServiceLogoutSPAFlow"
+```mdx-code-block
+<Tabs group="console-or-cli">
+  <TabItem value="console" label="Ory Console" default>
+```
 
-<SelfServiceLogoutSPAFlow />
+You can set a default return URL in the [Ory Console](https://console.ory.sh/projects/current/browser-redirects) on the **Browser
+Redirects** page using the **Post-Logout Redirect** value.
 
-The following scripts show you how to do the browser flow based on cURL. Prerequisites are a recent version of cURL and a
-registered account in Ory Identities.
+```mdx-code-block
+  </TabItem>
+  <TabItem value="cli" label="Ory CLI">
+```
 
-Before we start, we need to log in:
+You can set the default return URL after a logout using the [Ory CLI](../../../guides/cli/01_installation.mdx):
 
 ```shell
-# Username/email and password for an existing account
-username="myuser@ory.sh"
-password="mysecret"
-
-# We use this cookie jar to initiate the login flow
-cookieJar=$(mktemp)
-
-# Initialize the flow
-flow=$( \
-  curl -s -H "Accept: application/json"  --cookie $cookieJar --cookie-jar $cookieJar \
-    'https://playground.projects.oryapis.com/self-service/login/browser' \
-)
-
-# Get the action URL
-actionUrl=$(echo $flow | jq -r '.ui.action')
-
-# Get the CSRF Token
-csrfToken=$( \
-  echo $flow | \
-    jq -r '.ui.nodes[] | select(.attributes.name=="csrf_token") | .attributes.value' \
-)
-
-# Complete the login
-session=$( \
-  curl -s --cookie $cookieJar --cookie-jar $cookieJar -X POST \
-    -H "Accept: application/json" -H "Content-Type: application/json" \
-    --data '{ "identifier": "'$username'", "password": "'$password'", "method": "password", "csrf_token": "'$csrfToken'" }' \
-    "$actionUrl" \
-)
-
-echo $session | jq
-
-# Check the current user id
-curl -s --cookie $cookieJar --cookie-jar $cookieJar -H "Accept: application/json" \
-  https://playground.projects.oryapis.com/sessions/whoami | \
-  jq -r ".id"
+ory patch identity-config {project-id} \
+   --replace '/selfservice/flows/logout/after/default_browser_return_url="https://example.org"'
```

Comment:
It's better to add an intro sentence b4 the tabs: 

`To set the default post-logout return URL:`

in console tab: 

`Go to **Browser Redirects** and set the URL in the **Post-Logout Redirect** field.`

in CLI tab: 

`Run this command:`